### PR TITLE
turn MissingPartitions into ordered list

### PR DESCRIPTION
**Pull request by joesis**
So we can easily spot which partititions are missing.

### CODE REVIEW

File: zeno-cli/zeno-cli.go [hunk 1]
```diff
@@ -520,8 +525,8 @@ func printQueryStats(stderr io.Writer, md *common.QueryMetaData) {
 	fmt.Fprintf(stderr, "# As Of:      %v\n", md.AsOf.In(time.UTC).Format(time.RFC1123))
 	fmt.Fprintf(stderr, "# Until:      %v\n", md.Until.In(time.UTC).Format(time.RFC1123))
 	fmt.Fprintf(stderr, "# Resolution: %v\n", md.Resolution)
-	fmt.Fprintln(stderr, "\n")
+	fmt.Fprint(stderr, "\n")
```
Comment (joesis) [inline zeno-cli/zeno-cli.go]: Drive-by. Compiler complains about the extra linefeed.

File: web/index.go [hunk 2]
```diff
@@ -275,7 +275,7 @@ var indexHTML = []byte(`
 	        <span class="glyphicon {{#if running}}glyphicon-refresh glyphicon-spin{{else}}glyphicon-play{{/if}}" aria-hidden="true"></span> Run Now!
 	      </button>
 			  {{#if !running}}
-	        {{#if error}}<span class="error">Error: {{ error }}</span>{{elseif result}}<span class="summary">Queried: <b>{{ date }}</b>&nbsp;&nbsp;Partitions: <b>{{ result.Stats.NumSuccessfulPartitions }} / {{ result.Stats.NumPartitions }}</b>{{#if result.Stats.MissingPartitions }}&nbsp;&nbsp;Missing Partitions: <b>{{ result.Stats.MissingPartitions }}{{/if}}&nbsp;&nbsp;Complete To: <b>{{ completeUpTo }}</b></span>{{/if}}
+		{{#if error}}<span class="error">Error: {{ error }}</span>{{elseif result}}<span class="summary">Queried: <b>{{ date }}</b>&nbsp;&nbsp;Partitions: <b>{{ result.Stats.NumSuccessfulPartitions }} / {{ result.Stats.NumPartitions }}</b>{{#if result.Stats.MissingPartitions }}&nbsp;&nbsp;Missing Partitions: <b>{{ result.Stats.MissingPartitions }}{{/if}}&nbsp;&nbsp;Complete To: <b>{{ completeUpTo }}</b></span>{{/if}}
```
Comment (joesis) [inline web/index.go]: Nothing changed actually. ractive shows string array exactly in comma seperated format.

File: zeno-cli/zeno-cli.go (hunk 1 above)
Comment (oxtoacart) [reply zeno-cli/zeno-cli.go]: I guess without the `ln` we need to add an extra line feed to keep the formatting, right?

**Comment by oxtoacart** [comment]
LGTM other than the possible formatting change in `zeno-cli`.

**Comment by oxtoacart** [comment]
LGTM!

**Comment by joesis** [comment]
Thanks @oxtoacart !
